### ISSUE TITLE
we can get 1.20+?

### ISSUE THREAD

**Issue by mitat52** [latibro/Enhanced-Villagers-Mod#15]
mod is really good and still needs develop but new versions would be good.

**Comment by latibro** [comment]
You are right this was an idea i started out, and then got distracted (sorry :-) ), but really want to find time to modding again.

And it's always easiest to mod to the latest version (which was 1.19 when I did this first version), and then port to others version when you have some thing working.

Thanks for you feedback - that is really good fuel for my motivation to continue :-)

Feel free to join my Discord server to discuss ideas of where this mod could move.
https://discord.gg/W6UxS3vDfb

**Comment by mitat52** [comment]
no need sorry :) about your motivation, thats good to hear :D I hope you can keep your motivation for everything in your life. also you're one person to do mod or more? and if u will do new versions, new versions can come in how much time?

**Comment by latibro** [comment]
Hi @mitat52, i have upgraded it to 1.21 now - it might also work for 1.20, i haven't tried it, but feel free to try it out and let me know if it also works for 1.20.

**Comment by mitat52** [comment]
wow thats nice! im gonna try and can left feedback that like basic looking test if you would like too because as a very small animator and deals with pc, doing a minecraft modpack and learning mods behaviour for now, making for future and this mod is on my eyes for mod's behaviour. good luck for develop but whats went wrong for forge😅 also i'll write to other opened issue about mod's "demo" so trailer of mod when you complete it or for develop i can take somethings about that if you would accept, i would be happy. its can be good for your motivation too, sounds good?

**Comment by latibro** [comment]
@mitat52 here is the story of forge [https://neoforged.net/news/theproject/](https://neoforged.net/news/theproject/) - TLDR they split to Forge og NeoForge, and Forge wasn't ready with 1.21 when I started the upgrade (they might be by now, haven't checked). But the NeoForge version might work for Forge, i haven't tested it.

Join my Discord server [https://discord.latibro.com](https://discord.latibro.com) and let's have a talk about how you could help me out, help is always nice :smile:
